src/components/ChatEntry.jsx

```diff
@@ -1,20 +1,39 @@
+import { useState } from 'react';
+import PropTypes from 'prop-types';
 import './ChatEntry.css';
+import TimeStamp from './TimeStamp';
+
+const ChatEntry = ({sender, body, timeStamp, onLike, onUnlike}) => {
```

Comment:
```suggestion
const ChatEntry = ({ sender, body, timeStamp, onLike, onUnlike }) => {
```

Should have white spaces around curly braces when using destructuring syntax.

I'd also expect `ChatLog` to pass the value of `liked` to each `ChatEntry` so that this component knows if `liked` is `true` or `false` to set a heart color.